### PR TITLE
I <3 Circles

### PR DESCRIPTION
**Pull request by marto55**

### CODE REVIEW

**Review by GerganaRoeva — APPROVED** [review]
Много красиво с много любов писано! 6/6

File: homework_03/13_martin_georgiev/circlesHW.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+class Point:
+    def __init__(self, x, y):
+        assert isinstance(x, (int, float))
+        assert isinstance(y, (int, float))
+        self.x = x
+        self.y = y
+
+class Circle:
+    def __init__(self, O, r):
```
Comment (Lilly7777) [inline homework_03/13_martin_georgiev/circlesHW.py:17]: Change the name of the circle's center to a name that is more intuitive.

File: homework_03/13_martin_georgiev/circlesHW.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+        if self.r > circle2.r:
+            big_r = self.r
+            small_r = circle2.r
+        else:
+            small_r = self.r
+            big_r = circle2.r
+
+        # if distance > self.r + circle2.r
+        if distance - (self.r + circle2.r) > 0.001:
```
Comment (Lilly7777) [inline homework_03/13_martin_georgiev/circlesHW.py:37]: The difference between AB and the sum of the radiuses may be 0.00001. It is enough for AB to be greater than the sum.

File: homework_03/13_martin_georgiev/test_circlesHW.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
+from circlesHW import RelativePosition, Point, Circle
```
Comment (Lilly7777) [inline homework_03/13_martin_georgiev/test_circlesHW.py:1]: I have no remarks on the tests. Well done!

**Review by Lilly7777 — APPROVED** [review]

**Pull request merged**